packages/flutter_tools/lib/src/ios/simulators.dart

```diff
@@ -420,20 +417,13 @@ class IOSSimulator extends Device {
   }
 
   Future<void> _setupUpdatedApplicationBundle(covariant BuildableIOSApp app, BuildInfo buildInfo, String mainPath) async {
-    await sideloadUpdatedAssetsForInstalledApplicationBundle(buildInfo, mainPath);
```

Comment:
This doesn't appear to do anything

Reply:
Who wrote it? Can we ask?

Reply:
I lost the thread around https://github.com/flutter/flutter/pull/14016 . It seems like it was used pre-Dart 2 to prepare the FLX bundle, and has persisted since then